api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiAuthenticator.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+/**
+ * This service perform authentication to MPI server. It is a Spring managed bean which is configured
+ * in moduleApplicationContext.xml.
+ */
+public interface MpiAuthenticator {
```

Comment:
My question is: does _general_ code need to call authenticator? Wouldn't this be something that is purely called from within the OpenEMPI implementation of our MPI interfaces?